daphne/src/lib.rs

```diff
@@ -225,6 +236,25 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_batch_duration`, `min_batch_interval_start`,
+// and `max_batch_interval_end` should be reflected to the DAP spec.
```

Comment:
Hmm, arguably this is not needed. Would you mind bringing this up in an issue? https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/new I'm wondering if anyone has thought about this yet.

Reply:
Since this is a Daphne-specific implementation detail, I've swapped the TODO with a note regarding whether to consider putting this into the spec or not.